src/System.Runtime.Serialization.Xml/tests/project.json

```diff
@@ -32,13 +32,9 @@
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.7": {}
   },
   "supports": {
-    "coreFx.Test.netcoreapp1.0": {},
-    "coreFx.Test.net46": {},
-    "coreFx.Test.net461": {},
-    "coreFx.Test.net462": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.netcoreapp1.1": {}
```

Comment:
I added "coreFx.Test.net463" back. For UWP, what's the framework we should support? @weshaggard 

---
In reply to: [82297411](https://github.com/dotnet/corefx/pull/12378#discussion_r82297411) [](ancestors = 82297411)